backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewEdgeService.java

```diff
@@ -36,10 +43,21 @@
  */
 public class ViewEdgeService {
 
+    /**
+     * The default separator used when printing a set of Guard Expressions for a TransitionUsage label
+     */
+    private static final String GUARD_EXPRESSION_SEPARATOR = "& ";
```

Comment:
Where did you find in the specification that `&` should be used to separate guard expressions?

Reply:
This part of the code comes from the state transition view module (org.eclipse.syson.diagram.statetransition.view.services.StateTransitionViewEdgeService) where the guard expression is also displayed as part of the Transition edge label.
Currently, the state transition view diagram does not share its content with other diagrams (not even with the General view diagram), I have extracted this service in the common module as it should be used in several diagrams.